osu.Game/Screens/Edit/Components/Menus/EditorMenuBar.cs

```diff
@@ -14,19 +17,56 @@ namespace osu.Game.Screens.Edit.Components.Menus
 {
     public partial class EditorMenuBar : OsuMenu
     {
+        private const float heading_area = 114;
+
         public EditorMenuBar()
             : base(Direction.Horizontal, true)
         {
             RelativeSizeAxes = Axes.X;
 
             MaskingContainer.CornerRadius = 0;
-            ItemsContainer.Padding = new MarginPadding { Left = 100 };
+            ItemsContainer.Padding = new MarginPadding { Left = heading_area };
         }
 
         [BackgroundDependencyLoader]
-        private void load(OverlayColourProvider colourProvider)
+        private void load(OverlayColourProvider colourProvider, TextureStore textures)
         {
             BackgroundColour = colourProvider.Background3;
+
+            TextFlowContainer text;
+
+            AddRangeInternal(new[]
+            {
+                new Container
+                {
+                    RelativeSizeAxes = Axes.Y,
+                    Width = heading_area,
+                    Padding = new MarginPadding(8),
+                    Children = new Drawable[]
+                    {
+                        new Sprite
+                        {
+                            Size = new Vector2(26),
+                            Anchor = Anchor.CentreLeft,
+                            Origin = Anchor.CentreLeft,
+                            Texture = textures.Get("Icons/Hexacons/editor"),
+                        },
+                        text = new TextFlowContainer
+                        {
+                            Anchor = Anchor.CentreRight,
+                            Origin = Anchor.CentreRight,
+                            AutoSizeAxes = Axes.Both,
+                        }
+                    }
+                },
+            });
+
+            text.AddText("osu!", t => t.Font = OsuFont.TorusAlternate);
+            text.AddText("editor", t =>
+            {
+                t.Font = OsuFont.TorusAlternate;
+                t.Colour = colourProvider.Highlight1;
+            });
```

Comment:
This header is also present in the skin editor which may be confusing. I would rename it to `beatmap/skin editor`.

I also don't get why the prefixing of "osu!" in certain places is a thing (maybe just in the design mockups?). We had `osu!news` in the website for a while and `osu!chat` during the chat redesign PR before I renamed it in https://github.com/ppy/osu/pull/18433 because of localisation.

Reply:
It's supposed to give emphasis to certain game areas or features. Think of it as visual flair, not helpful text that needs to be localised.

In this case, I'd argue the header here should not be localised if it is to remain at all. As in, it should either be present as `osu!editor` on all languages, or be removed completely.

Reply:
So does this fall in the "Services and applications" listed in <https://osu.ppy.sh/wiki/en/Brand_identity_guidelines#osu!>? If so, it is a proper noun and shouldn't be localised.

And as a proper noun, `osu!editor` can only be the beatmap editor or skin editor and not both. That is where my problem is.

I also feel like we are returning back to using `osu!` prefixes (like `osu!direct` but is now `beatmap listing`).

Reply:
yeah well osu!direct is another point of contention. most people know it by this branding so we may actually return to it.